src/main/java/sg/ncl/MainController.java

```diff
@@ -2809,6 +2814,12 @@ public String internetRequest(@PathVariable String teamName,
         return "redirect:/experiments";
     }
 
+    @RequestMapping("/web_ssh/access_gateway")
+    public String webAccessGateway(Model model, HttpSession session) throws WebServiceRuntimeException {
+        HttpEntity<String> request = getDeterUid(model, session);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "request". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "request" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)